FAQ.md

```diff
@@ -1,14 +1,49 @@
 
-If you're using Bukkit, it's likely that you'll also need to install `Vault` alongside your chat/tab plugin, don't forget it!
+***
 
-### How do I get permissions to sync across multiple servers
-Connect each LuckPerms installation to the same MySQL/MongoDB server. You can use `/luckperms sync` to pull the latest changes from the database. You can also [setup a Messaging Service](https://github.com/lucko/LuckPerms/wiki/Network-Installation#messaging-service) to have your changes sync instantly between servers.
+### Why are permissions not working?
+First, *make sure LuckPerms is your only permission plugin!* We have had many issues where people will install LuckPerms and not remove another permission manager like PermissionsEx. You *must* remove any other permission plugins otherwise LuckPerms will not be used for permission checking. The only exception is when you are performing a migration, in which case you must have both permission plugins installed but please remember to remove the old plugin afterwards.
+
+If LuckPerms is your only permission plugin, then the issue could be that you have not added the correct permissions for what you require. Always make sure to consult the documentation of the plugins you are using to see what the available permissions are. If the documentation is incomplete, then you can use [Verbose mode](https://github.com/lucko/LuckPerms/wiki/Verbose) to see which permissions are being checked in real time and what value LuckPerms returns for the user that is being checked.
+
+***
+
+### Why are prefixes/suffixes not working?
+Most of the time this isn't actually an issue with LuckPerms itself but here are a few troubleshooting tips:
+
+- LuckPerms is *not* a chat formatter and another plugin is required to format the chat or tab list to include prefixes and suffixes.
+  - If you are running a Bukkit based server (CraftBukkit, Spigot, Paper) then you also need to install [Vault](https://dev.bukkit.org/bukkit-plugins/vault/).
+  - Some popular chat formatting plugins are listed [here](https://github.com/lucko/LuckPerms/wiki/Prefixes,-Suffixes-&-Meta#displaying-prefixes-and-suffixes).
+- Firstly, you should check if a prefix/suffix is actually set with LuckPerms - run `/lp user <user> info` and see if the prefix/suffix is displayed correctly. If so, great! LuckPerms has done its job and now it's up to your chat/tab/whatever formatter to display it correctly. If not, you need to [set a prefix or suffix](https://github.com/lucko/LuckPerms/wiki/Prefixes,-Suffixes-&-Meta).
+- If you are using Essentials, make sure you are running [EssentialsX and EssentialsXChat](https://ci.ender.zone/job/EssentialsX/). The X is important! Other versions of Essentials may not work correctly with LuckPerms.
+  - Run `/ess version` for a quick rundown on your Essentials setup, this also reports whether Vault is installed and which permission plugin you are running - preferably LuckPerms!
```

Comment:
```suggestion
  - Run `/ess version` for a quick rundown on your Essentials setup, this also reports whether Vault is installed and which permission plugin you are running - preferably LuckPerms! If the command only returns a message like "Reloaded Essentials [version]" then you're either using Essentials or an unsupported, outdated version of EssentialsX.
```

Reply:
I committed the change but adjusted it a little bit.